content/blog/tidyselect-1-2-0/index.Rmd

```diff
@@ -0,0 +1,158 @@
+---
+output: hugodown::hugo_document
+
+slug: tidyselect-1-2-0
+title: tidyselect 1.2.0
+date: 2022-10-14
+author: Lionel Henry and Hadley Wickham
+description: >
+    tidyselect 1.2.0 hit CRAN last week and includes a few updates to the syntax of selections in tidyverse functions.
+
+photo:
+  url: https://unsplash.com/photos/xZxZxiceD8s
+  author: Laura Gilchrist
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package]
+tags: [lifecycle, tidyselect]
+---
+
+<!--
+TODO:
+* [ ] Look over / edit the post's title in the yaml
+* [ ] Edit (or delete) the description; note this appears in the Twitter card
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+[tidyselect](https://tidyselect.r-lib.org/) 1.2.0 hit CRAN last week and includes a few updates to the syntax of selections in tidyverse functions like `dplyr::select(...)` and `tidyr::pivot_longer(cols = )`.
+
+tidyselect is a low-level package that implements the backend for selection contexts in tidyverse functions. A selection context is an argument like `cols` in `pivot_longer()` or a set of arguments like `...` in `select()` [^1]. In these special contexts, you can use a dialect of R that helps you create a selection of columns. You can select multiple columns with `c()`, a range of columns with `:`, and complex matches with selection helpers such as `starts_with()`. Under the hood, this selection syntax is interpreted and processed by the tidyselect package.
+
+[^1]: If you are wondering whether a particular argument supports selections, look in the function documentation. Arguments tagged with `<tidy-select>` implement the selection dialect. By contrast, arguments tagged with `<data-masking>` only allow to refer to data frame columns directly.
+
+In this post, we'll cover the most important [lifecycle changes](https://lifecycle.r-lib.org/articles/stages.html) in the selection syntax that tidyverse users should know about. You can see a full list of changes in the [release notes](https://tidyselect.r-lib.org/news/index.html#tidyselect-120). We'll start with a quick recap of what it means in practice for a feature to be deprecated or soft-deprecated.
```

Comment:
Maybe emphasise that this blog post is most important if you're writing a package that uses dplyr or tidyr?